pkg/build/api/v1beta3/types.go

```diff
@@ -385,6 +390,9 @@ type BuildRequest struct {
 
 	// Revision is the information from the source for a specific repo snapshot.
 	Revision *SourceRevision `json:"revision,omitempty"`
+
+	// Image is the image that triggered this build.
+	Image string `json:"image,omitempty"`
```

Comment:
BuildRequest is different structure. This one is internally named DockerImageReference (see [here](https://github.com/openshift/origin/blob/master/pkg/build/api/types.go#L282)) it's used to pass the name of the output registry the resulting image should be pushed too. Though looking at v1beta3 conversion, I can't see this field being assigned to aforementioned, generally speaking it looks like it's ignored.